### DISCUSSION TITLE
Can't find python?

### DISCUSSION THREAD

**Discussion by avengerofsquids** [nianeyna/ao3downloader#86] [Q&A] [answered]
I've tried uninstalling and reinstalling multiple versions of python (3.10.8 and 3.9.0) but neither of them have worked. I also tried deleting and redownloading the ao3downloader with no success. When I try to run the "ao3downloader.cmd" I get this error: 
![image](https://user-images.githubusercontent.com/132944516/236911085-a7cb22b6-1256-4ec5-83c0-503bf32a9112.png)
When I press any key to continue, it disappears, and none of my attempts to fix it have worked. I am on Windows 10. When I search for Python in my start menu it shows up and has no issues. I currently have 3.9.0 installed. Apologies if this is an obvious solution that I'm missing, I'm new to Python.

**Comment by verotheelf** [comment, marked as answer]
I haven't had a windows computer in a long time, but it seems your computer isn't properly pointing to the python install. If you type `python --version` into the powershell, what does it output? 

One common issue appears to be the need to turn off any "App installer" toggles for python that you see in "Manage App Execution Aliases" (can find this setting by looking it up in the search bar). 

If that doesn't work I would guess it's an issue with your PATH, basically your computer's directions for where to find python. In the install setup there should've been an option to "Add Python x to PATH", not sure if it's checked by default. I would reinstall and make sure that box is checked. If that still doesn't fix it, you can manually add the path, but someone more familiar with windows would have to talk you through that or you can probably find instructions online.

Another thing you can try is installing python directly from the website instead of through the Microsoft store. The link for the downloads can be found in the install instructions for ao3downloader

**Reply by avengerofsquids** [reply]
Thank you! My computer didn't cooperate with the "read me" nianeyna mentioned, so I must've missed it. I turned off the app installer toggles and reinstalled python to double check that the PATH option was enabled, and it's working now!

**Comment by nianeyna** [comment]
>One common issue appears to be the need to turn off any "App installer" toggles for python that you see in "Manage App Execution Aliases" (can find this setting by looking it up in the search bar).

Yes, this is likely the issue. There is a link in the Troubleshooting section of the ao3downloader readme that goes into more detail about this.
